default.build

```diff
@@ -170,9 +170,12 @@
   <target name="build_windows10" description="Build Windows 10 UAP" depends="clean">
     <if test="${os == 'Win32NT'}">
       <if test="${file::exists('c:\Program Files (x86)\MSBuild\Microsoft\WindowsXaml\v14.0\Microsoft.Windows.UI.Xaml.CSharp.targets')}">
-        <exec program="Protobuild" commandline="-generate WindowsUAP" />
-        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUAP.sln /t:Clean /p:Configuration=Release /p:Platform="Any CPU"' />
-        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUAP.sln /t:Build /p:Configuration=Release /p:Platform="Any CPU"' />
+        <exec program="Protobuild" commandline="-generate WindowsUniversal" />
+        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUniversal.sln /t:Clean /p:Configuration=Release /p:Platform="Any CPU" /p:ResolveNuGetPackages=false' />
+        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUniversal.sln /t:Build /p:Configuration=Release /p:Platform="Any CPU" /p:ResolveNuGetPackages=false' />
```

Comment:
> Yeah... but you have to agree how ridiculous that is getting. You need to figure out a better way to track modules than littering EXEs around.

I don't see it as ridiculous at all.  It's an intentional design of Protobuild that each module contains it's own implementation of the project generator so that:

* Newer versions of Protobuild don't break existing modules
* Users don't need to install anything

Unless you go looking in the package directories, you don't see this stuff anyway (it's like going into the `packages/` folder and complaining there's hundreds of `.nuspec` or `.nupkg` files there, of course there are, that's where the package information is stored)